### ISSUE TITLE
[BLUS30145] Prototype, graphical glitch at start screen covers screen in green glitching effect

### ISSUE THREAD

**Issue by Omechengineeru** [RPCS3/rpcs3#14013]
### Quick summary

instead of the normal red start screen I get this
![image](https://github.com/RPCS3/rpcs3/assets/92702685/6014b193-c1ea-4fbe-90b0-ba5b875429f9)
as well as the other noticable feature of insanly low framerates

### Details

this should be what the start screen looks like normally 
![image](https://github.com/RPCS3/rpcs3/assets/92702685/00a41971-744b-4b74-9feb-237f12547a60)

### Attach a log file

[RPCS3.log](https://github.com/RPCS3/rpcs3/files/11729094/RPCS3.log)

### System configuration

AMD Ryzen 5 3600 6-core CPU, GTX 1650s GPU

**Comment by Omechengineeru** [comment]
I have very limited experience with emulators so pls ELI5

**Comment by CorpseSlayer** [comment]
no has any update on this issue? i am pretty surprise that RPCS3 so advance on running some games wont run this game like others.

**Comment by vithusel** [comment]
Had this issue as well this morning. Mostly goes away after entering the game. 

Glad it runs. Pc version is horribly optimised and will crash on any system with more than 4 cores.

**Comment by ddmdareal** [comment]
This still seems to be an issue, happens on some cutscenes including the main "press start" and main menu screen, but sometimes it doesn't display garbage, maybe they are rendered real time? Attached a log and RSX capture.
[BLES00269_20240429134006_capture.rrc.gz](https://github.com/RPCS3/rpcs3/files/15148622/BLES00269_20240429134006_capture.rrc.gz)
[RPCS3.log](https://github.com/RPCS3/rpcs3/files/15148627/RPCS3.log)

**Comment by lastavenger2004** [comment]
I also have the problem too.

Here's the log
[RPCS3.log.gz](https://github.com/user-attachments/files/17688826/RPCS3.log.gz)

The problem is that the rendered videos cannot be played

Note: I had to compress the video a bit too much, so I could send it here.
https://github.com/user-attachments/assets/0949eb87-9d39-406c-8983-07b4c3e7f612
